config/webpack.config.js

```diff
@@ -18,6 +19,13 @@ var production =
   process.env.RAILS_ENV === 'production' ||
   process.env.NODE_ENV === 'production';
 
+var publicPath;
+if (production) {
+  publicPath = process.env.ASSET_PATH || '/webpack/';
+} else {
+  publicPath = process.env.ASSET_PATH || `http://${process.env.HOSTNAME}:${devServerPort}/webpack/`;
```

Comment:
do you really need the entire url? I would prefer to avoid hardcoding the path if we URL scheme

Reply:
Unfortunately we really need the entire url. We have to pass the dev server port. Without that webpack fetches dynamic imports from wrong url.